mu4/scenes/playback/internal/playbackconfiguration.cpp

```diff
@@ -0,0 +1,33 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#include "playbackconfiguration.h"
+
+using namespace mu::scene::playback;
+
+bool PlaybackConfiguration::isPlayElementOnClick() const
```

Comment:
Why do you save the fact of playing an element on click in the settings? Or does it return the ability to play the element on click?

Reply:
second
This is the setting - whether the element should be played on click